cmd/install/assets/hypershift_operator.go

```diff
@@ -909,6 +913,18 @@ func (o HyperShiftEtcdPriorityClass) Build() *schedulingv1.PriorityClass {
 	}
 }
 
+func HypershiftOperatorPriorityClass() *schedulingv1.PriorityClass {
```

Comment:
diverging from the
```
type HyperShiftEtcdPriorityClass struct{}
func (o HyperShiftEtcdPriorityClass) Build() *schedulingv1.PriorityClass {
...
```

pattern here.  Would like to keep them all the same.

Reply:
Id honestly rather change them all. That kind of pattern of defining an unused static object to then call a method on it is reserved for java code

Reply:
> Id honestly rather change them all. That kind of pattern of defining an empty static object to then call a method on it is reserved for java code

shots fired 🤣